metacat-schema-converters/build.gradle

```diff
@@ -0,0 +1,16 @@
+
+apply plugin: 'java'
+apply plugin: 'groovy'
+
+dependencies {
+    compile project(':metacat-common')
+    compile project(':metacat-type-converters')
+    compile("org.apache.hive:hive-metastore:1.2.1")
+    compile "com.google.guava:guava:${guava_version}"
+    compile "com.google.inject:guice:${guice_version}"
+    compile 'com.netflix.archaius:archaius-core:0.6.5'
+    compile 'com.netflix.servo:servo-core:0.8.3'
+    compile 'org.apache.pig:pig:0.14.0'
```

Comment:
Do we really need to rely on pig? that seems like a very large dependency that we may want to make pluggable in the design... like you can add type converter modules optionally but those modules should pull in these types of dependencies not the core module